ansible/roles/cluster/defaults/main.yml

```diff
@@ -13,3 +13,4 @@ cluster_pod_subnet: ""
 # Default etcd values, change these if you experience "leader changed" issues when running on a SD card
 cluster_etcd_heartbeat_interval: 100
 cluster_etcd_election_timeout: 1000
+kube_proxy: enabled
```

Comment:
would be good to have a comment in here for users to roughly know why you'd have this disabled.

Reply:
i'm unfamiliar with the requirements for cilium, so i am under the assumption that this is a requirement for it. if that is the case, then this var should be scoped as such. explicitly setting this var for later use could be misleading to some users. having this var set based on what CNI you're hoping to deploy would be the ideal situation, likely from a conditional import vars based on CNI selection from the user.

Reply:
It's not a requirement, but it would modify the installation options for cilium if you're not using kube-proxy.

Reply:
I would still prefer that the variable be scoped, or at the very least some comments be added to detail what you're enabling/disabling.